.github/workflows/phpstan.yml

```
@@ -0,0 +1,26 @@
+name: PHPStan
+
+on:
+  push:
+    paths:
+      - '**.php'
+      - 'phpstan.neon.dist'
+
+jobs:
+  phpstan:
+    name: phpstan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+          coverage: none
+
+      - name: Install composer dependencies
+        uses: ramsey/composer-install@v2
+
+      - name: Run PHPStan
+        run: ./vendor/bin/phpstan --error-format=github
```

I think it's better to be explicit in CI:s, so maybe add analyse here.
Replace with analyze depending on English/American (PHPStan supports both). 😅 

```suggestion
        run: ./vendor/bin/phpstan analyse --error-format=github
```